torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -736,34 +760,33 @@ class TensorExprKernel {
 
       case aten::cat: {
         return Compute(
-          "aten_cat",
-          texprDims(v),
-          [this, v](const std::vector<Var>& axes) {
-            Node* n = v->node();
-            auto inputs = n->inputs()[0]->node()->inputs();
-            size_t dim = n->inputs()[1]->node()->i(attr::value);
-
-            std::vector<Expr> new_axes(axes.begin(), axes.end());
-            Expr load = tensorOrConstant(inputs[0], new_axes);
-            size_t offset = bufferSizes(tensors_.at(inputs[0]->unique()))[dim];
-            new_axes[dim] = new_axes[dim] - IntImm::make(offset);
-
-            for (int ii = 1; ii < inputs.size(); ++ii) {
-              load = ifThenElse(
-                CompareSelect::make(axes[dim], IntImm::make(offset), kLT),
-                load,
-                tensorOrConstant(inputs[ii], new_axes)
-              );
-              offset += bufferSizes(tensors_.at(inputs[ii]->unique()))[dim];
+            "aten_cat", texprDims(v), [this, v](const std::vector<Var>& axes) {
```

Comment:
Are these just formatting changes?  Please revert formatting changes that aren't part of your functional diff.